src/core/CMakeLists.txt

```diff
@@ -800,7 +800,10 @@ set(TEST_FILES
     tests/unittests/volumesequenceutils-tests.cpp
     tests/unittests/zip-test.cpp
 )
+# Default inviwo unit tests run after InviwoApplication initialization
 ivw_add_unittest(${TEST_FILES})
+# Tests for initialization/deinitialization of InviwoApplication 
+add_subdirectory(tests/unittests/without-initialization)
```

Comment:
I think it would be better to keep this outside of core, and also have it graded by IVW_TEST_INTEGRATION_TESTS
and not IVW_TEST_UNIT_TESTS

hence I would add the test here instead. 
https://github.com/inviwo/inviwo/blob/a832ca089466d8bf50c221eed7974cbe10708398/CMakeLists.txt#L109

that way we might also enable more modules and so on to increase the test surface in the future? 

I would then more the files to /tests/recycle-inviwoapplication/ to something like that.
without-initialization does not seem very descriptive of the test per say...

Reply:
That is actually where I placed it first, but since the tests only dealt with core I thought it would be better to keep it there. 

I can move it and do renaming, but i will keep core as the only dependency for now. Another PR could in that case extend the testing to check more modules with singletons.

Reply:
yes, that way my thinking also

Eventually the goal would be that it would work with all modules.